packages/contracts/contracts/plugin/PluginRepo.sol

```diff
@@ -118,110 +168,106 @@ contract PluginRepo is
             revert InvalidPluginSetupInterface({invalidPluginSetup: _pluginSetup});
         }
 
-        uint256 currentVersionIndex = nextVersionIndex - 1;
```

Comment:
Does it bring value to spend gas in this condition? :thinking: 
- Maybe a developer wants to do a prerelease (but this should happen on testnet, not in prod)
- But maybe we need to ensure that nobody created a build on an empty release